lib/run/index.js

```diff
@@ -210,6 +210,7 @@ module.exports = function (options, callback) {
                         emitter.emit('assertion', (assertion.passed ? null : {
                             name: errorName,
                             index: assertion.index,
+                            test: assertion.name,
```

Comment:
This needs to be added as test. Isn’t it?

Reply:
Added